app/experimenter/nimbus-ui/src/components/PageEditOverview/index.tsx

```diff
@@ -3,35 +3,74 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { useCallback, useState } from "react";
-import { RouteComponentProps, useParams } from "@reach/router";
+import { navigate, RouteComponentProps, useParams } from "@reach/router";
 import AppLayoutWithSidebar from "../AppLayoutWithSidebar";
 import { useExperiment } from "../../hooks";
 import HeaderEditExperiment from "../HeaderEditExperiment";
 import PageExperimentNotFound from "../PageExperimentNotFound";
 import PageLoading from "../PageLoading";
 import FormOverview from "../FormOverview";
+import { useMutation } from "@apollo/client";
+import { UpdateExperimentInput } from "../../types/globalTypes";
+import { updateExperimentOverview_updateExperimentOverview as UpdateExperimentOverviewResult } from "../../types/updateExperimentOverview";
+import { UPDATE_EXPERIMENT_OVERVIEW_MUTATION } from "../../gql/experiments";
+import { SUBMIT_ERROR } from "../../lib/constants";
 
 type PageEditOverviewProps = {} & RouteComponentProps;
 
 const PageEditOverview: React.FunctionComponent<PageEditOverviewProps> = () => {
-  // TODO: EXP-462 Get this from constants / config loaded at app start?
-  const applications = ["firefox-desktop", "fenix", "reference-browser"];
+  const [updateExperimentOverview, { loading: mutationLoading }] = useMutation<
+    { updateExperimentOverview: UpdateExperimentOverviewResult },
+    { input: UpdateExperimentInput }
+  >(UPDATE_EXPERIMENT_OVERVIEW_MUTATION);
 
   const { slug } = useParams();
-  const { experiment, notFound, loading } = useExperiment(slug);
-  const [submitErrors /* setSubmitErrors */] = useState<Record<string, any>>(
-    {},
+  const { experiment, notFound, loading: experimentLoading } = useExperiment(
+    slug,
   );
+  const [submitErrors, setSubmitErrors] = useState<Record<string, any>>({});
 
-  const onFormSubmit = useCallback(() => {
-    console.log("SUBMIT TBD");
-  }, []);
+  const onFormSubmit = useCallback(
```

Comment:
We need a mock in Storybook for this not to error, though it would be also be nice to have a state besides "basic" to show an error state.

<img width="1229" alt="image" src="https://user-images.githubusercontent.com/13018240/97901163-67e27800-1d01-11eb-89f5-8a6fcf75fea0.png">

Reply:
Sure I can add that. FWIW I'm on the fence about this, because `FormOverview` does have its own stories (including with existing experiment data and error states), so we're really looking to demonstrate the page layout and not necessarily the form. But I can see how for completeness you should be able to successfully hit Save. Maybe we don't need all the error states though?

Reply:
Yeah, I feel like things under `Pages` should work somewhat well like clicking "Save" either sends a successful mutation or simply links to `PageEditOverview` (the latter probably being easier). It'd be nice for one of the pages to show what the error state looks like, anything extra is probably bonus points 🤷

Reply:
To recap our Slack discussion, it doesn't appear that there's a great approach here without pulling things apart a bit.

> clicking "Save" either sends a successful mutation

While it's possible to create a mock for this, the mutation input needs to match the form input, otherwise it won't get picked up. So a mock would immediately become invalidated once a user starts typing in any of the fields.

> like on the PageNew page, simply links to PageEditOverview

The Save button, by way of `FormOverview`, actually already links to this component! But I think Storybook intercepts this as already being on that story, so it does nothing and still results in the error being shown.

There might be other approaches that I'm not thinking of at the moment, but it sounds like we might want to follow up on this to avoid blocking? Is that correct?

Reply:
Oof, speaking of the PageNew story: I just noticed that it looks like this commit broke it.

* Before: https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/265a366028dce040318162dd5fa0459c45e41e69/nimbus-ui/index.html?path=/story/pages-new--basic

* After: https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/5797b0f1eaeb53adb68951e40cf0f2715cc0fe8f/nimbus-ui/index.html?path=/story/pages-new--basic